app/models/commissioning_document.rb

```diff
@@ -32,7 +32,7 @@ def document
   def filename
     return unless valid?
 
-    "Day1_#{request_type}_#{case_number}_#{subject_name}_#{timestamp}.#{mime_type}"
```

Comment:
Hi Andrew (@vertism), are you happy the removal of the timestamp won't cause a problem such as duplicate filename issues?

Reply:
It certainly could in theory. The request to change this came from the users, but I will check that they understand this risk. Thanks.